daemon/cmd/kube_proxy_replacement.go

```diff
@@ -307,6 +307,22 @@ func detectNativeDevices(strict bool) {
 // replacement after all devices are known.
 func finishKubeProxyReplacementInit(isKubeProxyReplacementStrict bool) {
 	if option.Config.EnableNodePort {
+		// Check whether the DirectRoutingDevice (if specified) is defined within devices
```

Comment:
Maybe it make sense to move the new code to `detectNativeDevices(...)` function, and then rename the function to `handleNativeDevices(...)` or so?

Reply:
don't have a strong opinion either way, but I pushed a change with the change above.